frontend/src/metabase/query_builder/actions/core/updateQuestion.ts

```diff
@@ -126,11 +128,7 @@ export const updateQuestion = (
       );
     }
 
-    // Sync card's parameters with the template tags;
-    if (isNewQuestionNative) {
-      const parameters = getTemplateTagParametersFromCard(newQuestion.card());
-      newQuestion = newQuestion.setParameters(parameters);
-    }
+    newQuestion = syncCardParametersWithTemplateTags(newQuestion);
```

Comment:
This is the exact same logic, just extracted into `syncCardParametersWithTemplateTags` so I can reuse it everywhere else including in the SDK.